src/control/cmd/dmg/pretty/pool.go

```diff
@@ -215,37 +211,43 @@ func printListPoolsRespVerbose(out io.Writer, resp *control.ListPoolsResp) error
 	return nil
 }
 
-func validateListPoolsResp(r *control.ListPoolsResp) error {
-	var numTiers int
-
-	for i, p := range r.Pools {
-		if p.UUID == "" {
-			return errors.Errorf("pool with index %d has no uuid", i)
-		}
-		if len(p.Usage) == 0 {
-			return errors.Errorf("pool %s has no usage info", p.UUID)
-		}
-		if numTiers != 0 && len(p.Usage) != numTiers {
-			return errors.Errorf("pool %s has %d storage tiers, want %d",
-				p.UUID, len(p.Usage), numTiers)
-		}
-		numTiers = len(p.Usage)
-	}
-
-	return nil
-}
-
 // PrintListPoolsResponse generates a human-readable representation of the
 // supplied ListPoolsResp struct and writes it to the supplied io.Writer.
 // Additional columns for pool UUID and service replicas if verbose is set.
-func PrintListPoolsResponse(o io.Writer, r *control.ListPoolsResp, v bool) error {
-	if err := validateListPoolsResp(r); err != nil {
+func PrintListPoolsResponse(o, e io.Writer, r *control.ListPoolsResp, v bool) error {
```

Comment:
naming nit - the io.Writer names aren't obvious. How about "out" and "errOut"?

Reply:
done